package.json

```diff
@@ -47,7 +47,7 @@
     "alpheios-experience": "github:alpheios-project/experience",
     "alpheios-fixtures": "github:alpheios-project/fixtures",
     "alpheios-inflection-tables": "github:alpheios-project/inflection-tables",
-    "alpheios-lexis-cs": "github:alpheios-project/lexis-cs",
+    "alpheios-messaging": "github:alpheios-project/alpheios-messaging#mss-service-split",
```

Comment:
I believe the branch would be removed before merging.

Reply:
Yes, I will remove it